src/createUiDefinition.json

```diff
@@ -57,8 +57,8 @@
                         "toolTip": "Initial username for InfluxDB admin user.",
                         "constraints": {
                             "required": true,
-                            "regex": "^(?!(?:adm|admin|audio|backup|bin|cdrom|crontab|daemon|dialout|dip|disk|fax|floppy|fuse|games|gnats|irc|kmem|landscape|libuuid|list|lp|mail|man|messagebus|mlocate|netdev|news|nobody|nogroup|operator|plugdev|proxy|root|sasl|shadow|src|ssh|sshd|staff|sudo|sync|sys|syslog|tape|tty|users|utmp|uucp|video|voice|whoopsie|www\\-data)$)\\w+$",
-                            "validationMessage": "Username must not be a reserved username"
+                            "regex": "^(?!(?:adm|admin|audio|backup|bin|cdrom|crontab|daemon|dialout|password|disk|fax|floppy|fuse|games|gnats|irc|kmem|landscape|libuuid|list|lp|mail|man|messagebus|mlocate|netdev|news|nobody|nogroup|operator|plugdev|proxy|root|sasl|shadow|src|ssh|sshd|staff|sudo|sync|sys|syslog|tape|tty|users|utmp|uucp|video|voice|whoopsie|www\\-data)$)\\w+$",
```

Comment:
This was kind of annoying when I was doing a demo. I think we can life the naming restriction for the InfluxDB username since there's no collision risk.

Reply:
Yep ,agree - removed validation constraint